### PR TITLE
Made changes

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# FortiSOAR Solution Pack: Automated Employee Onboarding
-
-[![Solution Pack Video](https://img.youtube.com/vi/ytVDEOY432A/0.jpg)](https://www.youtube.com/embed/ytVDEOY432A)
```
Comment (cs-akash-jarad) [inline README.md:3]: need to correct the video, starting bits are not accurate . the installation process will be different in the solution pack case.

File: docs/README.md [hunk 2]
```diff
@@ -1,80 +0,0 @@
...
-- MS Exchange/SMTP Connectors configured with your MS Exchange parameters (To receive instruction emails and send notifications)
-- AD Connector configured with your DC parameters (To run AD related actions)
-- MySQL/SSH server(s) to act as resources
-
-> All Credentials used with the above connectors must have the required privileges to perform the actions they are meant to do (add user, disable user...etc)
-
-> You can customize the use case for your environment by adding 3 playbooks for each service: Provision, Deprovision , Management the latter is only used as a wrapper for the first two. Open any examples of the existing services to see how is it done
-
-
-### 3) Installation/Deployment:
```
Comment (cs-akash-jarad) [inline docs/README.md:47]: install process is not correct .

File: README.md (hunk 1 above)
Comment (cs-suyog-jinturkar) [reply README.md:3]: Ankit, we will need to ask original submitter of this PR for this change.

Comment (cs-ankit-gupta) [reply README.md:3]: @ftnt-cse, can we make video more presentable?

File: docs/README.md (hunk 2 above)
Comment (cs-ankit-gupta) [reply docs/README.md:47]: Docs for this SP is getting changed, will pull once changes are made

**Pull request closed without merging**